lib/src/presentation/view/card_page.dart

```diff
@@ -0,0 +1,207 @@
+
+class CardPage extends StatefulWidget {
+  final HearthstoneCard hearthstoneCard;
+
+  const CardPage({
+    required this.hearthstoneCard,
+    super.key,
+  });
+
+  @override
+  State<CardPage> createState() => _CardPageState();
+}
+
+class _CardPageState extends State<CardPage> {
+  int _counter = counterInitialValue;
+
+  void _incrementCounter() {
+    setState(() {
+      _counter++;
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return SafeArea(
+      child: Scaffold(
+        appBar: AppBar(
+          title: const Text(
+            appBarCardPageTitle,
+            style: TextStyle(
+              fontSize: appBarCardPageTitleFontSize,
+            ),
+          ),
+          backgroundColor: Colors.black,
+        ),
+        body: SingleChildScrollView(
+          child: Center(
+            child: Stack(
+              children: [
+                Container(
+                  height: backgroundImageContainerHeight,
+                  width: double.infinity,
+                  decoration: const BoxDecoration(
+                    image: DecorationImage(
+                      image: AssetImage(
+                        assetPathBackgroundImage,
+                      ),
+                      fit: BoxFit.cover,
+                    ),
+                  ),
+                ),
+                Column(
+                  children: <Widget>[
+                    Padding(
+                      padding: const EdgeInsets.only(
+                        left: cardTextPaddingLeftTopRight,
+                        top: cardTextPaddingLeftTopRight,
+                        right: cardTextPaddingLeftTopRight,
+                        bottom: cardTextPaddingBottom,
+                      ),
+                      child: Column(
+                        children: <Widget>[
+                          VisibilityText(
+                            cardAttribute:
+                                '$cardName  ${widget.hearthstoneCard.name}',
+                            visibleCondition: true,
+                          ),
+                          VisibilityText(
+                            cardAttribute:
+                                '$cardText ${widget.hearthstoneCard.text}',
+                            visibleCondition:
+                                widget.hearthstoneCard.text != null,
+                          ),
+                          VisibilityText(
+                            cardAttribute:
+                                '$cardHealth ${widget.hearthstoneCard.health}',
+                            visibleCondition: widget.hearthstoneCard.health !=
+                                emptyValueIntHearthstoneCardFactory,
+                          ),
+                          VisibilityText(
+                            cardAttribute:
+                                '$cardType ${widget.hearthstoneCard.type}',
+                            visibleCondition: true,
```

Comment:
I think instead to call visibleCondition is better approach to call like isVisible

Reply:
or visible

Reply:
Done